src/ai/backend/manager/models/alembic/versions/85984c98b90f_update_endpoint_and_routing_table.py

```diff
@@ -6,21 +6,31 @@
 
 """
 
+import enum
+
 import sqlalchemy as sa
 from alembic import op
 from sqlalchemy.dialects import postgresql
 
 from ai.backend.common.types import ClusterMode
 from ai.backend.manager.models.base import GUID, URLColumn
-from ai.backend.manager.models.routing import RouteStatus
 
 # revision identifiers, used by Alembic.
 revision = "85984c98b90f"
 down_revision = "857bdec5abda"
 branch_labels = None
 depends_on = None
 
-routestatus_choices = [v.value for v in RouteStatus]
+
+class LegacyRouteStatus(enum.Enum):
+    HEALTHY = "healthy"
+    UNHEALTHY = "unhealthy"
+    TERMINATING = "unhealthy"
```

Comment:
Is `TERMINATING` also `unhealthy`?

Reply:
In #1278, the enum type was defined with duplicate values by mistake.
And in #2191, the enum type has been corrected by adding a new "terminating" value.

The migration script `38741c` made in #2191 raises an error saying 'terminating value already exists' because `RouteStatus` is imported directly in `85984`, which leads to create the `routestatus` enum with correct values.
So, I leave the typo by purpose.